### DISCUSSION TITLE
Image and label orientation using Orientationd transform

### DISCUSSION THREAD

**Discussion by Suchi97** [Project-MONAI/MONAI#3998] [General]
Hi Team, 

We used a sample image and the corresponding label (BTCV dataset : img0001.nii.gz and label0001.nii.gz) to test the **orientationd transform.** 

Below transforms are used for the above purpose.
```
transforms = Compose([LoadImaged(keys=["image", "label"]), EnsureChannelFirstd(keys=["image","label"]),
                           ScaleIntensityRanged(keys=["image"], a_min=-175, a_max=250, b_min=0, b_max=1.0, clip=True),
                           Orientationd(keys=["image","label"], axcodes='RAS'),    # axcodes = RAS, RPI, LPI, LAS
                           EnsureTyped(keys=["image","label"])
                          ])
```
When we used the orientationd, we changed the axcodes values and for each axcodes, we obtained the below results.

**RAS**
![image](https://user-images.githubusercontent.com/25792114/159957918-f9918659-cfe9-486c-836a-08d5fa018eda.png)

**RPI**
![image](https://user-images.githubusercontent.com/25792114/159958370-a38eb829-b12b-464b-b9d8-f615d34e3e7b.png)

**LPI**
![image](https://user-images.githubusercontent.com/25792114/159958606-ebfd0336-8653-46c3-a9df-27f91ecf3cbf.png)

**LAS**
![image](https://user-images.githubusercontent.com/25792114/159958937-1068a76a-fc2e-4d31-88c2-58cd44597276.png)

From the above images, we can see that the images and labels are not aligned properly.

Can you please help me to solve this issue.
Thanks in advance

**Comment by rijobro** [comment]
Can you:
1. Show what the images look like with just `LoadImage(["image", "label"])`? 
2. Print `image_meta_dict` and `label_meta_dict` (of just the difference between them)?
3. Show what code you're using for the matplotlib?
